### ISSUE TITLE
The results of instance retrieval

### ISSUE THREAD

**Issue by Baymine** [lixiaotong97/DSU#5]
Hi, thanks for sharing your code. 

I run instance retrieval on my machine. On the first test, the mAP is 24.9%. But in the following test, the mAP gradually decreased. On the last test, the mAP is only 1.2%.

 Do you have any ideas about why this phenomenon happened? Thank you very much!

**Comment by GoodMan0** [comment]
@Baymine Hi, did you fix the problem? I have the same issue.

**Comment by lixiaotong97** [comment]
Hi, could you please provide more information such as the running environment, training script, and the loss value for convergence. Moreover, did you modify anywhere of the code?

Best regards.

**Comment by freshman97** [comment]
> @Baymine Hi, did you fix the problem? I have the same issue.

Hi, I also get the same problem...

**Comment by lixiaotong97** [comment]
> 

There is a simple way to try to solve this. You can use the original repo MMT (https://github.com/yxgeee/MMT), and comment this [line](https://github.com/yxgeee/MMT/blob/057e1ea5d3054c9d7e5fa72c727298d8e4c5f668/mmt/trainers.py#L40). Then, you can add DSU module into the resnet.py and should get similar results.  

I run without problems in my environment, so if you find any reson to cause it, please kindly let me know.

Best regards.

**Comment by feimo49** [comment]
@freshman97 Hi, did you fix the problem? I have the same issue.

**Comment by lixiaotong97** [comment]
You can try the above solution.

**Comment by freshman97** [comment]
> @freshman97 Hi, did you fix the problem? I have the same issue.

Nope. After commenting on that line it seems we still have the decreasing mAP problem. But you can try the implementation setup of [MixStyle](https://github.com/KaiyangZhou/mixstyle-release) and add the DSU module.
